packages/test/test-version-utils/src/testApi.ts

```diff
@@ -103,78 +118,214 @@ const DataRuntimeApi = {
 	},
 };
 
-export function getLoaderApi(baseVersion: string, requested?: number | string): typeof LoaderApi {
+async function loadLoader(
```

Comment:
nit: Should there functions be called `loadAndGet<X>`?

Reply:
good point. I think it's actually better in the other direction--we never use the return value of these. I'll keep the name but make them all void return :)